storage/transfer.js

```diff
@@ -0,0 +1,453 @@
+    // information.
+    if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+      // Scopes can be specified either as an array or as a single,
+      // space-delimited string.
+      authClient = authClient.createScoped([
+        'https://www.googleapis.com/auth/cloud-platform'
+      ]);
+    }
+    callback(null, authClient);
+  });
+}
+// [END auth]
+
+// [START create_transfer_job]
+/**
+ * Review the transfer operations associated with a transfer job.
+ *
+ * @param {string} srcBucketName The name of the source bucket.
+ * @param {string} destBucketName The name of the destination bucket.
+ * @param {string} [description] Optional description for the new transfer job.
+ * @param {function} callback The callback function.
+ */
+function createTransferJob (srcBucketName, destBucketName, date, time, description, callback) {
+  if (!srcBucketName) {
+    return callback(new Error('"srcBucketName" is required!'));
+  } else if (!destBucketName) {
+    return callback(new Error('"destBucketName" is required!'));
+  }
+
+  var startDate = moment(date, 'YYYY/MM/DD');
+  var transferTime = moment(time, 'HH:mm');
+
+  auth(function (err, authClient) {
+    if (err) {
+      return callback(err);
+    }
+
+    var transferJob = {
+      projectId: process.env.GCLOUD_PROJECT,
+      status: 'ENABLED',
+      transferSpec: {
+        gcsDataSource: {
+          bucketName: srcBucketName
+        },
+        gcsDataSink: {
+          bucketName: destBucketName
+        },
+        transferOptions: {
+          deleteObjectsFromSourceAfterTransfer: false
+        }
+      },
+      schedule: {
+        scheduleStartDate: {
+          year: startDate.year(),
+          month: startDate.month() + 1,
+          day: startDate.date()
+        },
+        startTimeOfDay: {
+          hours: transferTime.hours(),
+          minutes: transferTime.minutes()
+        }
+      }
+    };
+
+    if (description) {
+      transferJob.description = description;
+    }
+
+    storagetransfer.transferJobs.create({
+      auth: authClient,
+      resource: transferJob
+    }, function (err, transferJob) {
+      if (err) {
+        return callback(err);
+      }
+
+      console.log('Created transfer job: %s', transferJob.name);
+      return callback(null, transferJob);
+    });
+  });
+}
+// [END create_transfer_job]
+
+// [START get_transfer_job]
+/**
+ * Get a transfer job.
+ *
+ * @param {string} jobName The name of the transfer job to get.
+ * @param {function} callback The callback function.
+ */
+function getTransferJob (jobName, callback) {
+  if (!jobName) {
+    return callback(new Error('"jobName" is required!'));
+  }
+
+  auth(function (err, authClient) {
+    if (err) {
+      return callback(err);
+    }
+
+    storagetransfer.transferJobs.get({
+      auth: authClient,
+      projectId: process.env.GCLOUD_PROJECT,
+      jobName: jobName
+    }, function (err, transferJob) {
```

Comment:
This is standard Node.js format. If I moved the function onto the next line, I'd also have to move the opening curly brace on line 137 to the next line, but I don't do that unless lines are exceeding 80 characters.